forge-webhook-parser.py

```diff
@@ -26,16 +29,35 @@ class PayloadFields:
     default_clone_path: Optional[Path] = None
 
 
-PAYLOAD_FILE = "./github-payload.json"
-
 parser = argparse.ArgumentParser()
 parser.add_argument(
-    "-f", "--file", help="webhook payload file to parse", default=PAYLOAD_FILE
+    "-f", "--file", help="webhook payload file to parse", default=glob.glob("*.json")
```

Comment:
The `default` unnecessarily complicates your implementation. If you remove it here, you won't later have to check if `args.file` is a list and so on.

I guess we could move it elsewhere but I'd probably rather not have it at all because scanning the entire directory can be slow, picking the first matching file is not very deterministic and can confuse people by accidentally parsing the wrong JSON, etc.

Then you can say that `-f` and `-u` are mutually exclusive and that one of them is required
https://docs.python.org/3/library/argparse.html#mutual-exclusion
(the third example)

Reply:
thanks! this is a good suggestion. the `required` flag removes the need for the try / except block too.